packages/code-block/partials/clipboard-button/index.tsx

```diff
@@ -45,6 +50,10 @@ function ClipboardButton({ className, getText }: ClipboardButtonProps) {
   // reset to the default appearance so that it's clear
   // the "Copy" button can be used again
   useEffect(() => {
```

Comment:
We wait till the `copiedState` has changed before calling the callback.

This way, the callback has richer information, as it provides a `copiedState`, rather than only reporting that the copy button was clicked (with an unknown outcome) as we'd get if we hooked into the copy button's `onClick`.